test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -859,6 +860,53 @@ TEST(nntrainer_LossLayer, setCost_02_n) {
   EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
 }
 
+TEST(nntrainer_ActivationLayer, init_01_1) {
+  int status = ML_ERROR_NONE;
+  nntrainer::ActivationLayer layer;
+  status = layer.initialize(false);
+
+  EXPECT_EQ(status, ML_ERROR_NONE);
+}
+
+TEST(nntrainer_ActivationLayer, setType_01_p) {
+  nntrainer::ActivationLayer layer;
+  EXPECT_NO_THROW(layer.setActivation(nntrainer::ACT_RELU));
+  EXPECT_NO_THROW(layer.setActivation(nntrainer::ACT_SOFTMAX));
+  EXPECT_NO_THROW(layer.setActivation(nntrainer::ACT_SIGMOID));
+  EXPECT_NO_THROW(layer.setActivation(nntrainer::ACT_TANH));
+}
+
+TEST(nntrainer_ActivationLayer, setType_02_n) {
+  nntrainer::ActivationLayer layer;
+  EXPECT_THROW(layer.setActivation(nntrainer::ACT_UNKNOWN), std::runtime_error);
+}
+
+TEST(nntrainer_ActivationLayer, forward_backward_01_p) {
+  int status = ML_ERROR_NONE;
+  int batch = 3;
+  int channel = 1;
+  int height = 1;
+  int width = 10;
+  std::vector<float> answer = {0, 0, 0, 0, 0, 0.1, 0.2, 0.3, 0.4, 0.5,
```

Comment:
where is this used?

Reply:
It is not currently used, I'd better delete this. Thanks!